libs/components/pages/src/lib/modules/link-as/link-as.pipe.ts

```diff
@@ -0,0 +1,39 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'linkAs',
+})
+export class LinkAsPipe implements PipeTransform {
+  public transform(
+    value: unknown,
```

Comment:
It seems a little odd to accept `item` or `item.permalink` here and then have to use `uknown` as the type. Can you not always accept `item` as `SkyActionHubNeedsAttention`? That way you can check for `permalink` in the function and avoid all the bracket notation and `'prop' in value` checks.

Reply:
💡 Updated.